docs/structured-streaming-programming-guide.md

```diff
@@ -1496,9 +1496,9 @@ Additional details on supported joins:
 
 - Joins can be cascaded, that is, you can do `df1.join(df2, ...).join(df3, ...).join(df4, ....)`.
 
-- As of Spark 2.3, you can use joins only when the query is in Append output mode. Other output modes are not yet supported.
+- As of Spark 2.4, you can use joins only when the query is in Append output mode. Other output modes are not yet supported.
```

Comment:
For now, it's `2.4` because this patch will be backported to `branch-2.4`. We will update this later when we prepare 3.0.0.